test/unit/compiler/MemorySymbolTableUnitTest.scala

```diff
@@ -48,7 +48,7 @@ class MemorySymbolTableUnitTest extends AnyFunSuite with BeforeAndAfter {
   val shape = new ShapeDeclaration("example", 4,1,
     new PrefixInvocation("example", 4,2, "this is my shape name", prefix), null)
 
-  before {
+  after {
```

Comment:
Was any problem with the original before impl?

Reply:
yes, I will document everything later on in the issue, this is still WIP

Reply:
okay thanks!

Reply:
I have updated the comment